src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -274,7 +274,7 @@ describe('Test suite for mappings on existing concepts', () => {
     const instance = wrapper.find('EditConcept').instance();
     instance.updateEventListener({ target: { tabIndex: 0, name: INTERNAL_MAPPING_DEFAULT_SOURCE, value: '' } });
     instance.updateEventListener(event);
-    expect(instance.state.mappings[0].to_concept_code).not.toEqual(null);
```

Comment:
You are not to change this. The code is not to be auto-generated, but the field is not to be empty as well. The user is to type of paste the code

Reply:
I have sorted this issue

Reply:
I have sorted this issue

Reply:
Also on the other comment, when a user tries to enter a source that is not the default CIEL, an empty text field is provided with the place holder `To concept code` where the code is supposed to be entered. This is also shown in the video recoding of after the fix.